test/recognizer/accounts_test.exs

```diff
@@ -94,10 +94,14 @@ defmodule Recognizer.AccountsTest do
     end
 
     test "registers users with a hashed password" do
-      {:ok, user} = Accounts.register_user(params_for(:user))
+      {:ok, user} =
+        :user
+        |> params_for(%{email: "TEST@Example.com"})
```

Comment:
I updated the create and update test to use wildly cased emails to assert it's downcased properly.